cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/KafkaCruiseControlServlet.java

```diff
@@ -194,7 +211,7 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response) t
    */
   @Override
   protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
-    ACCESS_LOG.info("Received {}, {} from {}", urlEncode(request.toString()), urlEncode(request.getRequestURL().toString()),
+    ACCESS_LOG.info("Received {}, {} from {}", request.toString(), request.getRequestURL().toString(),
```

Comment:
Same here. We have to urlEncode the request for logging.

Reply:
changing